domain/src/main/kotlin/no/nav/su/se/bakover/domain/behandling/Avsluttet.kt

```diff
@@ -0,0 +1,9 @@
+package no.nav.su.se.bakover.domain.behandling
+
+/**
+ * At noe er avsluttet vil si at den ikke kan endres. Eksempler kan være at man ferdig behandler noe, og den skal
+ * bli låst.
+ *
+ * Eksempel er [no.nav.su.se.bakover.domain.vedtak.Vedtak]
+ */
+interface Avsluttet
```

Comment:
Mulig å begynne legge på noen få felter på disse 2. id: UUID og opprettet: Tidspunkt. Man kan på sikt legge inn de nullables man trenger og, for å slippe when(avsluttet) og when(avbrutt).